rero_ils/modules/stats/api/report.py

```diff
@@ -294,7 +295,7 @@ def _circulation_config(self, trigger):
             # 0 if not period else 1 as a boolean is an integer
             'query': lambda: (
                 s := RecordsSearch(index=LoanOperationLog.index_name)[:0]
-                .filter('term', organisation__value=self.org_pid)
+                .filter('terms', loan__item__library_pid=self.lib_pids)
```

Comment:
The library filter should generally be on transaction library, except for requests indicators (item owning library).

Reply:
After some discussions, the current approach is correct.